doc/en/components/general-getting-started.md

```diff
@@ -288,19 +316,20 @@ npm run build
 <!-- end: WebComponents -->
 
 <!-- Angular, React -->
-# Getting Started With {ProductName}
 
 ## Updating Existing App
 
 If you want to use {ProductName} in an existing {Platform} CLI project (one that you have from before). We have you covered! All you have to do is execute these commands:
 
 ```cmd
+npm install --save {PackageCommon}
 npm install --save {PackageCharts} {PackageCore}
 npm install --save {PackageExcel} {PackageCore}
 npm install --save {PackageGauges} {PackageCore}
 npm install --save {PackageGrids} {PackageCore}
 npm install --save {PackageMaps} {PackageCore}
 npm install --save {PackageSpreadsheet} {PackageCore}
+npm install --save {PackageComponents}
```

Comment:
the "PackageComponents" variable does not exist in docsConfig. Did you forget to add it or did you mean to use "PackageCommon" which is already listed above?

Reply:
Removing it